fix: add the missing part of kong gateway vault's secrets versioning description (#6551)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/secrets-management/backends/hashicorp-vault.md

{vault://hashicorp-vault/hello/foo}
```

> If you've configured a secret value in multiple versions, like this:

Reviewer: ```suggestion
If you have configured a secret value in multiple versions:
```

```text
# Requires kv2 engine enabled
vault kv patch secret/hello foo=world2

======= Metadata =======
Key                Value
---                -----
created_time       2022-01-16T01:40:03.740833Z
custom_metadata    <nil>
deletion_time      n/a
destroyed          false
version            2
```

> Access an older version of the secret will be like this:

Reviewer: ```suggestion
Access an older version of the secret like this:
```
